frontend/web/components/pages/FeaturesPage.js

```diff
@@ -313,13 +310,11 @@ const FeaturesPage = class extends Component {
                             'features',
                             featureLimitAlert.percentage,
                           )}
-                        {environmentMetricsEnabled && (
-                          <EnvironmentMetricsList
-                            environmentApiKey={environment?.api_key}
-                            forceRefetch={this.state.forceMetricsRefetch}
-                            projectId={projectId}
-                          />
-                        )}
```

Comment:
Just this one if you don't mind I think it's worth keeping it for now as a kill switch. CF the issue about the permissions thunk query.

The thing is that we have never gone through the V2 which is supposed to add some caching and optimization. The identity part might be really heavy so it can be handful to deactivate if it gets out of control for a heavy org